src/mixins/portfolio.js

```diff
@@ -84,12 +84,22 @@ export const createPortfolioMixin = ({
     nftClassListOfCreated() {
       return this.nftClassListMap?.created || [];
     },
+    nftClassMapOfWritingNft() {
+      const nftClassMapOfWritingNft = Array.from(this.allNFTClassMap.values())
+        .filter(nft => {
+          const classMetadata = this.getNFTClassMetadataById(nft.classId);
+          const isWritingNFT = checkIsWritingNFT(classMetadata);
+          return isWritingNFT;
+        })
+        .reduce((map, nft) => map.set(nft.classId, nft), new Map());
+      return nftClassMapOfWritingNft;
+    },
     nftClassMapOfOther() {
       const nftClassMapOfOther = Array.from(this.allNFTClassMap.values())
         .filter(nft => {
           const classMetadata = this.getNFTClassMetadataById(nft.classId);
           const isWritingNFT = checkIsWritingNFT(classMetadata);
-          const isLikecoinChainNFTBook = checkIsNftBook(classMetadata);
+          const isLikecoinChainNFTBook = checkIsNFTBook(classMetadata);
           return !isWritingNFT && !isLikecoinChainNFTBook;
         })
         .reduce((map, nft) => map.set(nft.classId, nft), new Map());
```

Comment:
We can build `nftClassMapOfOther` using `nftClassMapOfWritingNft` and `nftClassMapOfNFTBook`